pkg/backporter/backporter.go

```diff
@@ -398,12 +403,12 @@ func getClonesTemplateData(bugID int, client bugzilla.Client, allTargetVersions
 	clonedReleases := sets.NewString()
 	for _, clone := range clones {
 		clone := clone
-		if len(clone.TargetRelease) > 0 {
-			majorRelease, err := getMajorRelease(clone.TargetRelease[0])
+		if len(clone.TargetRelease) > 0 && clone.TargetRelease[0] != "---" {
```

Comment:
We might want to expose this to the user -- you can't clone because your bug is not targeting anything.

Reply:
Can we highlight the cell of the table in red or something?

Reply:
Mouseover text to tell you that you messed up

Reply:
I can add some JS for that, make it a UI validation instead of keeping it as an error message from the backend. 
Also this is being exposed to the end user in the form of an error message right now.
Should I proceed with adding the JS

Reply:
Its not possible to highlight based on content without javascript? It also needs to stay a validation on the backend, input from the frontend is not trust worthy

Reply:
Statically highlighting the cell would be enough IMO